prepare.py

```diff
@@ -2,7 +2,7 @@
 import shutil
 
 import patch
-
+print("debug: which patch library is loaded? " + patch.__file__)
```

Comment:
what's this for?

Reply:
I overlooked this one. I encountered an import issue, which stemmed from an improper Conda-env setup. I didn't like the replication of gh-action jobs across various operating systems, but it appears there's no alternative solution.